includes/Core/Consent_Mode/REST_Consent_Mode_Controller.php

```diff
@@ -190,6 +191,29 @@ protected function get_rest_routes() {
 					),
 				)
 			),
+			new REST_Route(
+				'core/site/data/consent-api-activate',
+				array(
+					array(
+						'methods'             => WP_REST_Server::EDITABLE,
+						'callback'            => function () {
+							require_once ABSPATH . 'wp-admin/includes/plugin.php';
+
+							$slug      = 'wp-consent-api';
+							$plugin    = "$slug/$slug.php";
+
+							$activated = activate_plugin( $plugin );
+
+							if ( is_wp_error( $activated ) ) {
+								return new WP_Error( 'invalid_module_slug', $activated->get_error_message() );
+							}
+
+							return new WP_REST_Response( array( 'success' => true ) );
+						},
+						'permission_callback' => $can_manage_options,
```

Comment:
Shouldn't this use `Permissions:UPDATE_PLUGINS`?

https://github.com/google/site-kit-wp/blob/91fa03474cd51224ac0d0abd452525e4e9ee59a7/includes/Core/Permissions/Permissions.php#L44

Reply:
Good catch, thanks.